content/docs/installation/continuous-deployment-and-gitops.md

```diff
@@ -112,3 +112,141 @@ Check the cert-manager logs for warnings and errors:
 ```bash
 kubectl logs -n cert-manager -l app.kubernetes.io/instance=cert-manager --prefix --all-containers
 ```
+
+
+## Using ArgoCD
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+### Pre-requisites
+Ensure the following are in place before proceeding:
+- A Kubernetes cluster
+- ArgoCD deployed on the Kubernetes cluster: [installation guide](https://argo-cd.readthedocs.io/en/stable/getting_started/)
+- Optional: A GitOps repository connected with ArgoCD: [setup guide](https://argo-cd.readthedocs.io/en/stable/user-guide/private-repositories/)
+
+### Setting up cert-manager
+1. Create an ArgoCD Application manifest file with the provided configuration to set up cert-manager.
+
+    ```yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: cert-manager
+      namespace: argocd
+      annotations:
+        argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+      finalizers:
+        - resources-finalizer.argocd.argoproj.io
+    spec:
+      destination:
+        namespace: cert-manager
+        server: https://kubernetes.default.svc
```

Comment:
I'm surprised that you have to declare the URL of the Kubernetes API server. Is it strictly necessary? If not, I suggest removing it to avoid distracting the reader.